pkg/controller/scyllacluster/resource_test.go

```diff
@@ -617,11 +612,18 @@ func TestStatefulSetForRack(t *testing.T) {
 			Spec: appsv1.StatefulSetSpec{
 				Replicas: pointer.Ptr(int32(0)),
 				Selector: &metav1.LabelSelector{
-					MatchLabels: newBasicStatefulSetLabels(),
+					MatchLabels: map[string]string{
+						"app":                          "scylla",
+						"app.kubernetes.io/managed-by": "scylla-operator",
+						"app.kubernetes.io/name":       "scylla",
```

Comment:
we might want to keep the selector labels minimal (none of them is technically needed/used; note we have ownerRefs)

Reply:
we might, but it's immutable so we cannot change it now.

Reply:
yep, let's keep it in mind if we ever go for cascade delete